README.md

```diff
@@ -58,17 +56,22 @@ const { EventHubClient, EventHubReceiver, EventPosition } = require('azure-event
 const client = EventHubClient.fromConnectionString('Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key', 'myeventhub');
```

Comment:
>Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key [](start = 52, length = 118)

in general, even for examples, I would not encourage to show even mock connection strings as hardcoded. suggest using env variables or something else. we did in the iot hub sdks and we've had remarks on that.

Reply:
cool.